doc/classes/VisualInstance.xml

```diff
@@ -62,6 +62,16 @@
 			The render layer(s) this [VisualInstance] is drawn on.
 			This object will only be visible for [Camera]s whose cull mask includes the render object this [VisualInstance] is set to.
 		</member>
+		<member name="sorting_offset" type="float" setter="set_sorting_offset" getter="get_sorting_offset" default="0.0">
+			The sorting offset used by this VisualInstance.
+			A higher sorting offset will make the VisualInstance draw on top of other [VisualInstance] that are otherwise positioned at the same spot.
+			Best used with [member sorting_use_aabb_center].
+		</member>
+		<member name="sorting_use_aabb_center" type="bool" setter="set_sorting_use_aabb_center" getter="is_sorting_use_aabb_center" default="true">
+			Use the AABB center for transparent object sorting.
+			Best turned off when using [member sorting_offset].
+			This is particularly useful when working with [Particles] and [CPUParticles] nodes.
```

Comment:
I would also remove most of new lines. Remember, each new line is a new paragraph in the rendered documentation. So it's better to keep things on the same line, especially when dealing with short sentences like that.